src/views/v2/components/loader.php

```diff
@@ -11,14 +11,17 @@
  *
  * @version TBD
  *
+
  */
 ?>
 <div
 	class="tribe-events-view-loader tribe-common-a11y-hidden"
 	role="alert"
 	aria-live="assertive"
 >
-	<div class="tribe-events-view-loader__spinner">
-		<?php echo esc_html( $text ); ?>
-	</div>
+		<div class="tribe-events-view-loader__dots tribe-common-c-loader">
```

Comment:
spacing is off here and below

Reply:
btw nice use of classes here :+1: really liking it.

Reply:
Thank you!